docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -0,0 +1,107 @@
+---
+title: Install
+sidebar_position: 2
+description: How to run a Subspace Network Farmer with Space Acres GUI Application
+keywords:
+    - Farmer
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Space Acres installation
+
+## Step 1: Download the Space-Acres Installer
+
+<DownloadButtons />
+
+## Step 2: Install Space-Acres
```

Comment:
We need to replace all instances of Space-Acres to Space Acres to be consistent.

Reply:
Fixed in https://github.com/subspace/subspace-docs/pull/448